Source/Core/Common/BitSet.h

```diff
@@ -163,5 +171,6 @@ class BitSet
 
 }
 
+typedef BS::BitSet<u8> BitSet8;
```

Comment:
Yeah, there was a different patch that added that for the same reason, so I'll add it to this one too